src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -12,6 +12,13 @@
     <TestsSuccessfulSemaphore>tests.passed</TestsSuccessfulSemaphore>
   </PropertyGroup>
 
+  <!-- In case that TestPath is not yet set, default it here -->
+  <PropertyGroup>
+    <TestTargetOutputRelPath Condition="'$(TestTargetOutputRelPath)'=='' And '$(TargetGroup)'!='' And '$(TestTFM)'!=''">$(TargetGroup).$(TestTFM)/</TestTargetOutputRelPath>
+    <TestTargetOutputRelPath Condition="'$(TestTargetOutputRelPath)'=='' And '$(TargetGroup)'=='' And '$(TestTFM)'!=''">default.$(TestTFM)/</TestTargetOutputRelPath>
+    <TestPath Condition="'$(TestPath)'==''">$(TestWorkingDir)$(OSPlatformConfig)/$(MSBuildProjectName)/$(TestTargetOutputRelPath)</TestPath>
```

Comment:
@weshaggard with this change I'll set TestPath in tests.targets in case it is not set yet. It does have a problem though, since I'm only setting it if it is not set before, so in WCF most likely this will already be set incorrectly. What approach do you like better?:
- One option is to remove the condition from this here, so that TestPath gets overriden as dir.props is imported first.
- Keep Condition here, and just coordinate with WCF repo so that they fix dir.props to remove the setting of TestPath since it's going to live here now.